lib/config.py

```diff
@@ -66,6 +66,9 @@ def parse_itemsdir(itemsdir, item_conf):
         if item_file.endswith(CONF_FILE) or item_file.endswith(YAML_FILE):
             try:
                 item_conf = parse(itemsdir + item_file, item_conf)
+            except UnicodeDecodeError as e:
+                logger.error("item file ({}) is not in UTF-8 format, please check character encoding at position {}! => ignoring config".format(item_file, e.start))
+                continue
```

Comment:
Shouldn't be the `parse()` function the better place for checking encoding errors? Otherwise we will also run into the mentionen problem when reading other configuration files (like `smarthome.conf` or `logic.conf`) which has no proper encoding.